### PR TITLE
Split pages and add footer

### PR DESCRIPTION
**Pull request by NadineGrosskreuz**
Split into two pages and added a footer with a navbar
User Story #13

### CODE REVIEW

File: src/components/UI/Footer/FooterNav.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const FooterNav = styled.nav`
+	background-color: #342293;
+	width: 100%;
+	bottom: 0;
+	right: 0;
+	left: 0;
+	position: fixed;
```
Comment (irenepapaspyratos) [inline src/components/UI/Footer/FooterNav.js:9]: I think with a width of 100% you don't need left/right:0

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "react": "18.1.0",
     "react-dom": "18.1.0",
     "react-hook-form": "^7.31.1",
+    "storybook-addon-next-router": "^4.0.0",
```
Comment (irenepapaspyratos) [inline package.json:25]: Is there a special reason, that you did not install this addOn as a developer dependency? 🤔

**Review by irenepapaspyratos — APPROVED** [review]
👌

File: src/components/UI/Footer/FooterNav.js (hunk 1 above)
Comment (NadineGrosskreuz) [reply src/components/UI/Footer/FooterNav.js:9]: You are right, I removed it!

File: package.json (hunk 2 above)
Comment (NadineGrosskreuz) [reply package.json:25]: This is what npm install did. Would you recommend putting it in the devDependencies?

Comment (irenepapaspyratos) [reply package.json:25]: Yeah, npm install puts it in the regular dependencies but with -D or --save-dev it puts the addOn to the devDependencies.
If you ask me personally, I would put everything into the devs, that is not important for running the app itself  - but there might be different approaches and some addOns even NEED to be installed without dev. 
Just wanted to know, if there was a special thought behind. 🤓

**Pull request merged**